youths/apps.py

```diff
@@ -1,5 +1,11 @@
 from django.apps import AppConfig
 
+# from django.conf import settings
```

Comment:
well, I guess also this should be deleted

Reply:
🤦‍♂ yes